tests/test_custom_expert.py

```diff
@@ -0,0 +1,55 @@
+import os
+import pytest
+import torch
+import hivemind
+
+from typing import Optional
+from hivemind import RemoteExpert, background_server
+
+@pytest.mark.forked
+def test_custom_expert(port: Optional[int] = None, hid_dim=16):
+    with background_server(
+        expert_cls='perceptron', num_experts=2, device='cpu',
+        hidden_dim=hid_dim, num_handlers=2, no_dht=True,
+        custom_module_path=os.path.join(os.path.dirname(__file__), 'custom_networks.py')) as (server_endpoint, _):
+
+        expert0 = RemoteExpert('expert.0', server_endpoint)
+        expert1 = RemoteExpert('expert.1', server_endpoint)
+
+        for step in range(1, 4):
+            batch = torch.randn(step, hid_dim)
```

Comment:
1) Why is batch size step-dependent?
2) Why do steps go from 1 to 4?
3) Isn't just one step sufficient?

Reply:
Interpretation: it may be sufficient to check just two batch sizes: 1 and not_1

Reply:
Fixed